src/osemgrep/cli_ci/Ci_subcommand.ml

```diff
@@ -179,6 +179,7 @@ let scan_config_and_rules_from_deployment ~dry_run
       unique_id = Uuidm.v `V4;
       (* TODO: should look at conf.secrets, conf.sca, conf.code, etc. *)
       requested_products = [];
+      dry_run;
```

Comment:
maybe add a big TODO comment at the top of this file saying this is using the old way to request a scan and we should port the new code in scans.py that request a scan in one single POST request to /ap/cli/scans/

Reply:
Done